simple_certmanager/forms.py

```diff
@@ -15,6 +19,13 @@
 
 
 class SigningRequestAdminForm(forms.ModelForm):
+    certificate = forms.FileField(
+        required=False,
+        help_text="Upload the public certificate file here. "
+        "This will be used to verify the signature against the CSR "
+        "and create the certificate instance.",
+        label="Upload Certificate",
```

Comment:
1. user-facing text must be marked translatable
2. I think we can/should simplify the help text: "Upload the certificate file received from your certificate authority matching the CSR you sent them."